python/lsst/pipe/tasks/setPrimaryFlags.py

```diff
@@ -69,47 +75,58 @@ def run(self, sources, skyMap, tractInfo, patchInfo, includeDeblend=True):
         if includeDeblend:
             nChildKey = self.schema.find(self.config.nChildKeyName).key
 
-        # set inner flags for each source and set primary flags for sources with no children
-        # (or all sources if deblend info not available)
-        innerFloatBBox = Box2D(patchInfo.getInnerBBox())
+        if not self.isSingleFrame:
```

Comment:
Add comment explicitly stating this is the coadd case

Reply:
Comment added.